### PR TITLE
attempt to fix search lambda

### PR DESCRIPTION
**Pull request by nl0**

### CODE REVIEW

File: lambdas/search/index.py [hunk 1]
```diff
@@ -67,14 +67,14 @@ def lambda_handler(request):
     user_size = request.args.get('size', DEFAULT_SIZE)
     user_source = request.args.get('_source', [])
     # 0-indexed starting position (for pagination)
-    user_from = request.args.get('from', 0)
+    user_from = int(request.args.get('from', 0))
```
Comment (sir-sigurd) [inline lambdas/search/index.py:70]: Does it come as `str`?

**Review by sir-sigurd — COMMENTED** [review]
Tests are needed.

File: lambdas/search/index.py (hunk 1 above)
Comment (nl0) [reply lambdas/search/index.py:70]: i'm not sure, but since the invariant below is failing when `from=0`, i can deduce that it's not an `int`

**Comment by nl0** [comment]
> Tests are needed.

well, they are present (maybe not very high quality, i agree)

File: lambdas/search/index.py (hunk 1 above)
Comment (akarve) [reply lambdas/search/index.py:70]: It does. For some reason, the way we do query parameter processing the values always come as strings.

**Review by akarve — APPROVED** [review]

**Comment by sir-sigurd** [comment]
I think we should modify this somehow to ensure that it comes the same way as in prod:
https://github.com/quiltdata/quilt/blob/03695b07d0a779c9b072cfe9dc75cbd8608a6db3/lambdas/search/tests/test_search.py#L120
Something like `dict(urllib.parse.parse_qsl(urllib.parse.urlencode(query)))` could work.
We have similar `_make_event()` functions in tests of other lambdas, they should be modified as well.

**Pull request merged**